### PR TITLE
Implement/test LOLWUT command

### PR DESCRIPTION
**Pull request by brainix**
### Pull Request check-list

- [x] Does `$ tox` pass with this change (including linting)?
- [x] Does travis tests pass with this change (enable it first in your forked repo and wait for the travis build to finish)?
- [x] Is the new or changed code fully tested?
- [ ] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?

### Description of change

Implement/test [`LOLWUT`](https://redis.io/commands/lolwut) command

This is a lot of fun to play with:

```python
>>> from redis import Redis
>>> redis = Redis()
>>> print(redis.lolwut(5, 6, 7, 8).decode('utf-8'))
⣴⣶⣶⣶⣶⡆
⣿⣿⣿⣿⣿⡇
⠹⡿⠟⣿⡿⠃
⠀⠀⠀⠀⠀⠀
Georg Nees - schotter, plotter on paper, 1968. Redis ver. 6.0.10

>>> print(redis.lolwut(5, 6, 7, 8).decode('utf-8'))
⢰⣶⣶⣶⣶⡆
⢿⣿⣿⣿⣿⠁
⠸⡿⢿⠿⡿⠃
⠀⠀⠀⠀⠀⠀
Georg Nees - schotter, plotter on paper, 1968. Redis ver. 6.0.10

>>> print(redis.lolwut(5, 6, 7, 8).decode('utf-8'))
⢰⣶⣶⣶⣶⡆
⣸⣿⣿⣻⣿⡅
⠿⡿⠻⠿⠿⠁
⠀⠀⠀⠀⠀⠀
Georg Nees - schotter, plotter on paper, 1968. Redis ver. 6.0.10

>>>
```

### CODE REVIEW

**Comment by brainix** [comment]
@chayim - I'd originally opened this PR here: https://github.com/andymccurdy/redis-py/pull/1448, but I made a mess of that branch. So I recreated that PR here, putting the command and the test in the right places, and keeping the methods alphabetized. How does it look now? Thanks!

File: redis/commands.py [hunk 1]
```diff
@@ -504,6 +504,13 @@ def lastsave(self):
         """
         return self.execute_command('LASTSAVE')
 
+    def lolwut(self, *version_numbers):
+        "Get the Redis version and a piece of generative computer art"
+        if version_numbers:
```
Comment (chayim) [inline redis/commands.py:509]:
```suggestion
       return self.execute_command('LOLWUT VERSION', *version_numbers)
```

If version numbers is empty, it'll simply be concatenated as empty in the list. There's no need to use the if check.

File: redis/commands.py [hunk 2]
```diff
@@ -504,6 +504,13 @@ def lastsave(self):
         """
         return self.execute_command('LASTSAVE')
 
+    def lolwut(self, *version_numbers):
+        "Get the Redis version and a piece of generative computer art"
```
Comment (chayim) [inline redis/commands.py:508]:
```suggestion
        """Get the Redis version and a piece of generative computer art
        See: https://redis.io/commands/lolwut
        """
```

File: redis/commands.py (hunk 1 above)
Comment (brainix) [reply redis/commands.py:509]: I believe we need the `if` condition. If `version_numbers` is an empty list, then we don't want to include the string `VERSION` in the command. But if `version_numbers` contains values, then we want to include `VERSION` in the command.

If we pass an empty list along with `LOLWUT VERSION`, we get this error:

```python
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/rajiv.shah/Documents/Code/pottery/pottery/monkey.py", line 86, in lolwut
    return self.execute_command('LOLWUT VERSION', *version_numbers)
  File "/Users/rajiv.shah/Documents/Code/pottery/venv/lib/python3.9/site-packages/redis/client.py", line 901, in execute_command
    return self.parse_response(conn, command_name, **options)
  File "/Users/rajiv.shah/Documents/Code/pottery/venv/lib/python3.9/site-packages/redis/client.py", line 915, in parse_response
    response = connection.read_response()
  File "/Users/rajiv.shah/Documents/Code/pottery/venv/lib/python3.9/site-packages/redis/connection.py", line 756, in read_response
    raise response
redis.exceptions.ResponseError: value is not an integer or out of range
```

**Comment by brainix** [comment]
@chayim -- I believe I've addressed your feedback. 😅 How does this PR look now?

**Comment by chayim** [comment]
Looks great @brainix. Thanks for adding lolwut support. As a total aside - I really enjoyed playing with this, I think it's very funny that this is a thing. 

Part of #1546. Merging!

**Comment by brainix** [comment]
@chayim - Good call. I've decorated the unit test to be skipped for Redis < 5.0.0.

**Comment by chayim** [comment]
Amazing. Thanks again. Let's merge this in!

**Pull request merged**